xchange-core/src/test/java/org/knowm/xchange/dto/trade/UserTradeTest.java

```diff
@@ -50,7 +52,7 @@ public void testBuilder() {
   }
 
```

Comment:
please add one test that will allow us to check if CurrencyPair is serialized as a String:

```

  @Test
  public void testSerialize() throws IOException {
    UserTrade original = ...;
    String serialized = ObjectMapperHelper.toCompactJSON(original);
    assertThat(serialized).contains("\"currencyPair\":\"BTC/USD\"");
  }
```

Reply:
This is a valid extension to the same test. Done